authors/settings.py

```diff
@@ -81,11 +82,19 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv('database'),
+        'PORT': os.getenv('port'),
+        'USER': os.getenv('user'),
+        'PASSWORD': os.getenv('password'),
+        'HOST': os.getenv('host'),
     }
 }
 
+# Heroku DB config
+
```

Comment:
Thinking about this. Here's another approach
Since we are using the dj_database_url package for Heroku, we could also use it for setting up our database locally. In this case, we can update this to 

```python
DATABASES['default'] = dj_database_url.config(os.getenv[DATABASE_URL])
```

i.e for local development we can expose the database via `.env` as in `export DATABASE_URL=postgres://USER:PASSWORD@HOST:PORT/NAME`
With this, the `.envexample` will also change.

What do you think of this?

Reply:
That's actually a better approach. It does away with redundancy.